www/templates/logo.html

```diff
@@ -0,0 +1 @@
+<img src="img/rlogo.png" alt="rCredits logo" />
```

Comment:
For something as simple as a single image, a directive might be overkill. You can also consider ng-include. This is a helpful post: http://stackoverflow.com/a/24172025/2066866